optuna/pruners/successive_halving.py

```diff
@@ -128,6 +152,24 @@ def prune(self, study, trial):
 
             rung += 1
 
+    def _estimate_min_resource(self, trials):
+        # type: (Optional[List[FrozenTrial]]) -> None
+
+        if trials is None:
+            return
+
+        n_steps = [
+            t.last_step for t in trials
+            if t.state == TrialState.COMPLETE and t.last_step is not None
+        ]
+
+        if not n_steps:
+            return
+
+        # Get the maximum number of steps and divide it by 100.
+        last_step = max(n_steps)
+        self._min_resource = max(int(last_step / 100), 1)
```

Comment:
Can you simplify it as follows?
```suggestion
        self._min_resource = max(last_step // 100, 1)
```